lib/smack/SignedIntegerOverflowChecker.cpp

```diff
@@ -37,7 +37,7 @@ std::map<int, std::string> SignedIntegerOverflowChecker::INT_MIN_TABLE {
 
 bool SignedIntegerOverflowChecker::runOnModule(Module& m) {
   Function* va = m.getFunction("__SMACK_overflow_false");
-  Function* co = m.getFunction("__SMACK_check_overflow");
+  // Function* co = m.getFunction("__SMACK_check_overflow");
```

Comment:
@shaobo-he: can we completely remove this?

Reply:
No. there is a bug introduced by the commit: feffbf25eaa020e47e1adaa6f83839d4ad57f0d2. Can I use this branch to fix it?

Reply:
By all means, please do!

Reply:
Done.